pandas/io/formats/console.py

```diff
@@ -57,7 +57,7 @@ def in_interactive_session():
 
     def check_main():
         try:
-            import __main__ as main
+            import __main__ as main  # type: ignore
```

Comment:
This was ignored and won't be fixed as stated in mypy https://github.com/python/mypy/issues/658, though perhaps a better solution would be to just use `sys.stdin.isatty()` (follow up PR)

Reply:
IMO it would be nice to put a link the related mypy issue as comment too, for people in future who might be wonder why it's been ignored.
But l feel `isattay()` is indeed the better solution.